bedrock_plugin.py

```diff
@@ -71,7 +71,7 @@ def _authenticate(self):
 
 
 class CreatePipelineOperator(BaseOperator):
-    CREATE_PIPELINE_PATH = "{}/pipeline/".format(API_VERSION)
+    CREATE_PIPELINE_PATH = "/{}/pipeline/".format(API_VERSION)
```

Comment:
could you use urlparse for this

Reply:
it's an overkill here because we are only constructing the path parameter of a full url.